errors/deprecated-stream-type.md

```diff
@@ -0,0 +1,23 @@
+# Deprecated Stream Type
```

Comment:
(non-blocking) -- open to other opinions.

Can we put this on a branch for safe keeping? Feels weird to have it in the repo if it's never linked.

Might also want to put the commented code on a branch. Feels a little sloppy to have commented out code making it into `main`